src/features/favourites/Favourites.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import {inject, observer} from 'mobx-react';
+import {View, Text, Image} from 'react-native';
+
+import {Stores} from '@stores/stores';
+import {FoodsStore} from '@stores/foods';
+
+import {styles} from './styles/favourites';
+import {FlatList} from 'react-native-gesture-handler';
+import {Dish} from '@models/dish';
+
+interface Props {
+  dish: FoodsStore;
+}
+
+@inject(Stores.DishStore)
+@observer
+export class Favourites extends React.Component<Props> {
+  private get ListHeaderComponent() {
+    return (
+      <View style={styles.header}>
+        <Text style={styles.headerTitle}>Favourites</Text>
+      </View>
+    );
+  }
+
+  private get ListEmptyComponent() {
+    return (
+      <View>
+        <Text>Please add your favourite dishes</Text>
+      </View>
+    );
+  }
+  public render() {
+    return (
+      <View style={styles.container}>
```

Comment:
@DiamondRock597 Not sure but you can try to use only contentContainerStyle from FalatList, no need additional View component wrapper.